templates/topicedit.html

```diff
@@ -26,6 +26,10 @@ <h2>{{ if .Topic }}Edit{{ else }}New{{ end }} topic</h2>
         <input type="checkbox" id="is_sticky" name="is_sticky" value="1" {{ if .Topic }}{{ if .Topic.IsSticky }}checked{{ end }}{{ end }}>
         <label for="is_sticky">Sticky</label><br>
         {{ end }}
+        {{ if or .User.IsSuperAdmin .User.IsSuperMod }}
+        <input type="checkbox" id="is_closed" name="is_closed" value="1" {{ if .Topic }}{{ if .Topic.IsClosed }}checked{{ end }}{{ end }}>
+        <label for="is_closed">Close topic</label><br>
```

Comment:
Close topic is fine. If you feel the label Read-only is more clear, go for that.

Reply:
If a topic is closed, you shouldn't accept more comments to it. Are you doing that? It's OK to edit comments, but no new comments.